src/tests/ftest/util/job_manager_utils.py

```diff
@@ -473,6 +473,7 @@ def __init__(self, job, subprocess=False, mpi_type="openmpi"):
         self.tmpdir_base = FormattedParameter("--mca orte_tmpdir_base {}", None)
         self.args = BasicParameter(None, None)
         self.mpi_type = mpi_type
+        self.hostlist = FormattedParameter("-hosts {}", None)
```

Comment:
Why not use a hostfile?

Reply:
The host file would need to be created at script time and unique to each jobscript;  was just easier to use -hosts within the script.  Besides -hosts should be a param for mpirun